packages/utilities/src/customizations/Customizations.ts

```diff
@@ -1,13 +1,12 @@
-import { GlobalSettings } from './GlobalSettings';
-import { EventGroup } from './EventGroup';
+import { GlobalSettings } from '../GlobalSettings';
 
 // tslint:disable-next-line:no-any
-export type Settings = { [key: string]: any };
-export type SettingsFunction = (settings: Settings) => Settings;
+export type ISettings = { [key: string]: any };
```

Comment:
This is potentially a breaking change. These types should have been prefixed with an "I", but weren't, which was confusing to read. (e.g. what is a Settings class?)

Reply:
I've exported Settings/SettingsFunction as deprecated so anyone still importing it won't break.